download.html

```diff
@@ -80,8 +80,9 @@ <h2>Windows</h2>
 									</div>-->
 									<div class="6u 12u$(xsmall)">
 										<ul class="actions fit">
-											<li><a href="/cc/celestia-win-161" class="button special icon fa-download">Celestia 1.6.1 (x86, 33M)</a></li>
-											<li><a href="/cc/celestia-win-160" class="button icon fa-download">Celestia 1.6.0 (x86, 34M)</a></li>
+											<li><a href="https://github.com/CelestiaProject/Celestia/releases/download/1.6.2/celestia-1.6.2-win.exe" class="button special icon fa-download">Celestia 1.6.2 (x86/x64, 36M)</a></li>
```

Comment:
Changing "x64" to "x86/x64" makes this link overlap with the 1.6.1 link:
![celestiadownload](https://user-images.githubusercontent.com/56284207/99883820-c9f31680-2bf7-11eb-90a3-3c950023de73.png)

Reply:
i think it's better to clearly state that the installer supports both architectures. later we can fix css or buttons layout.